src/lib/utils.ts

```diff
@@ -27,8 +27,8 @@ export const filterAttributes = (attributes: string[], input: object | object[])
   return Array.isArray(input) ? input.map(filter) : filter(input);
 };
 
-export const transformPlaylistForClient = (playlist: Database.Playlist) =>
-  playlist.words?.map((word) => word.map((letter) => (letter === false ? null : letter)));
+export const transformPlaylistForClient = (playlist: Database.Playlist): (string | null)[][] =>
```

Comment:
as i was writing the tests, i noticed that we could add more typing to these utils so i updated them here and updated a couple of the calls in the `/playlists` apis.

Reply:
But this already has this type inferred? Did it not show up for you? May be something wrong with your setup if so.
![image](https://github.com/CSMA-Technology/blend-product-site/assets/10676387/edb07eda-ec3d-41c4-9d34-945c92596765)

Reply:
Oh just noticed the undefined, but that should be fixed with your nullish coalescing

Reply:
Also one more thing I just noticed in these functions, they should return the whole playlist. Since it is called `transformPlaylist...` it should be Playlist in, Playlist out

Reply:
Or we can rename it to `transformPlaylistWords...` but then it should just take the array of words as input IMO

Reply:
oh, true! I guess we only need the words, i'll refactor it to the latter

Reply:
Updated to:

![image](https://github.com/CSMA-Technology/blend-product-site/assets/6991957/a59c6fdc-a126-4bd9-be44-83b98ed564f9)